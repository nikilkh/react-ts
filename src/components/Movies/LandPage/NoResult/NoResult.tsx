import noResult from "../../../SVGs/noResult.svg";
import { ErrorHeadingStyle, NoResultStyle } from "./NoResult.styled";

export default function NoResult() {
  return (
    <>
      <NoResultStyle>
        <ErrorHeadingStyle>
          Sorry,there is no result for keyword you searched
        </ErrorHeadingStyle>
        <img src={noResult} alt="Sorry No Results"></img>
      </NoResultStyle>
    </>
  );
}
