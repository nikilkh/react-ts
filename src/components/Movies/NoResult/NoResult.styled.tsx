import styled from "styled-components";

export const NoResultStyle = styled.div`
  font-family: rubik;
  text-align: center;
  font-size: 32px;
  font-weight: 600;
  margin-top: 96px;
 margin: auto auto;
  width: 1440px;
  height: 1024px;

  @media only screen and (max-width: 375px) {
    width: 375px;
    height: 640px;
    margin-top: 24px;
  }
  @media only screen and (max-width: 425px) {
    width: 425px;
    height: 640px;
    margin-top: 24px;
  }
  @media only screen and (max-width: 600px) {
    width: auto;
    margin-top: 24px;
  }
  @media only screen and (max-width: 768px) {
    width: auto;
    margin-top: 28px;

  }
  @media only screen and (max-width: 1024px) {
    width: auto;
  }

  img {
    width: 620px;
    height: 447px;

    @media only screen and (max-width: 375px) {
      width: 240px;
      height: 224px;
      margin-top: 24px;
    }
    @media only screen and (max-width: 425px) {
      width: 410px;
      height: 224px;
      margin-top: 24px;
    }
    @media only screen and (max-width: 768px) {
    margin-top: 24px;

  }
    
  }
`;

export const ErrorHeadingStyle = styled.h2`
  margin-top: 0px;
  margin-bottom: 48px;

  @media only screen and (max-width: 375px) {
    font-size: 15px;
  }

  @media only screen and (max-width: 425px) {
    font-size: 20px;

  }
  @media only screen and (max-width: 768px) {
    font-size: 22px;
    

  }
`;
