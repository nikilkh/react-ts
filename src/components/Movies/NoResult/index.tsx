export * from "./NoResult";
export * from "./NoResult.styled";
