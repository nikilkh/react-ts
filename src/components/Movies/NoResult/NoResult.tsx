import { ErrorHeadingStyle, NoResultStyle } from "./NoResult.styled";
import noResult from "../../SVGs/noResult.svg";


export const NoResult = () => {
  return (
    <>
      <NoResultStyle>
        <ErrorHeadingStyle>
          Sorry,there is no result for keyword you searched
        </ErrorHeadingStyle>
        {/* <NoResult/> */}
        <img src={noResult} alt="Sorry No Results"></img>
      </NoResultStyle>
    </>
  );
}
