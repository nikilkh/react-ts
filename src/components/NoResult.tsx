import { NoResultStyle } from "../styles/NoResult.styled";
import noresult from "./noresult.svg";

export default function NoResult() {
  return (
    <>
      <NoResultStyle>
        <h2>Sorry,there is no result for keyword you searched</h2>
        <img src={noresult}></img>
      </NoResultStyle>
    </>
  );
}
