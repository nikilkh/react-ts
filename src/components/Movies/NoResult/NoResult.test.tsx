import { render } from "@testing-library/react";
import { NoResult } from "./NoResult";

describe("renders learn react link", () => {
  it("should render hello world", () => {
    const {container} = render(<NoResult/>);
    expect(container).toMatchSnapshot();
  });
});