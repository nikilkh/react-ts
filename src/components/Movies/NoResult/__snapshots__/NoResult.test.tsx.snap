// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link should render hello world 1`] = `
<div>
  <div
    class="sc-bczRLJ hfsExZ"
  >
    <h2
      class="sc-gsnTZi gBnMnc"
    >
      Sorry,there is no result for keyword you searched
    </h2>
    <img
      alt="Sorry No Results"
      src="noResult.svg"
    />
  </div>
</div>
`;
